content/integrations/AWS-Integration/AWS-IAM-Installation.md

```diff
@@ -10,53 +10,63 @@ weight: 2
 ---
 ## IAM Role Method
 
-Setting up an AWS integration via IAM Role is a two step process:
+Setting up an AWS integration via IAM Role is a three step process:
+
+1. Create a new AWS integration in Metricly.
+ - Optionally, define data filters for AWS elements to be included/excluded in Metricly using tags (key-value pair).
+2. Create an IAM role in your AWS Console.
+3. Add your IAM Role's ARN to your AWS integration in Metricly.
 
-Create a new AWS integration in Metricly using an IAM read-only role.
-Optionally, filter your AWS elements for inclusion in Metricly by creating or choosing an existing tag (key-value pair), then assigning that tag to the desired elements in AWS.
 
 {{% notice tip %}}
 If you already have an existing IAM role for Metricly but it does not include a policy for Cost Explorer, skip to the last section.
 {{% /notice %}}
 
-### Step 1: Create a new AWS integration
-1. From the top navigation menu, select Integrations.
-2. Click the Amazon Web Services card.
-3. Type a name for the new AWS integration. Ensure that Data Collection is selected.
-4. For AWS Authentication, select IAM Role.
+### Step 1: Create a new AWS integration in Metricly
+1. Login to Metricly and select the **Integrations** icon.
+![integrations-icon](/images/AWS-IAM-Installation/integrations-icon.png)
+2. Select the **Amazon Web Services** card.
+3. Provide a name for the new AWS integration.
```

Comment:
The landing page is only a blank config if no AWS integration has been configured. We should make a note to click the add integration button if an integrations exists.